ruma-common/src/push/predefined.rs

```diff
@@ -34,45 +34,45 @@ impl Ruleset {
     ///   user's ID (for instance those to send notifications when they are mentioned).
     pub fn server_default(user_id: &UserId) -> Self {
         Self {
-            content: set![PatternedPushRule::contains_user_name(user_id)],
+            content: set![ContentPushRule::contains_user_name(user_id)],
             override_: set![
-                ConditionalPushRule::master(),
-                ConditionalPushRule::suppress_notices(),
-                ConditionalPushRule::invite_for_me(user_id),
-                ConditionalPushRule::member_event(),
-                ConditionalPushRule::contains_display_name(),
-                ConditionalPushRule::tombstone(),
-                ConditionalPushRule::roomnotif(),
+                OverridePushRule::master(),
+                OverridePushRule::suppress_notices(),
+                OverridePushRule::invite_for_me(user_id),
+                OverridePushRule::member_event(),
+                OverridePushRule::contains_display_name(),
+                OverridePushRule::tombstone(),
+                OverridePushRule::roomnotif(),
             ],
             underride: set![
-                ConditionalPushRule::call(),
-                ConditionalPushRule::encrypted_room_one_to_one(),
-                ConditionalPushRule::room_one_to_one(),
-                ConditionalPushRule::message(),
-                ConditionalPushRule::encrypted(),
+                UnderridePushRule::call(),
+                UnderridePushRule::encrypted_room_one_to_one(),
+                UnderridePushRule::room_one_to_one(),
+                UnderridePushRule::message(),
+                UnderridePushRule::encrypted(),
             ],
             ..Default::default()
         }
     }
 }
 
 /// Default override push rules
-impl ConditionalPushRule {
+impl OverridePushRule {
     /// Matches all events, this can be enabled to turn off all push
     /// notifications other than those generated by override rules set by the user.
     pub fn master() -> Self {
-        Self {
+        OverridePushRule(ConditionalPushRule {
```

Comment:
```suggestion
        Self(ConditionalPushRule {
```

Reply:
I tried that, but apparently it is not a valid syntax. :woman_shrugging:

Reply:
Ok, I just re-tried and it works, my bad.